pkg/util/camel/camel_runtime_catalog.go

```diff
@@ -95,9 +95,9 @@ func (c *RuntimeCatalog) HasArtifact(artifact string) bool {
 		if c.Runtime.Provider == v1.RuntimeProviderQuarkus {
 			a = "camel-quarkus-" + a
 		} else {
-			a = "camel-" + a
 		}
 	}
+	a = "camel-" + a
```

Comment:
You should revert only this line, otherwise it will fail to set the dependency name for the component